docker/example-nginx-api.conf

```diff
@@ -10,6 +10,7 @@ server {
   proxy_buffer_size   256k;
 
   location / {
+    client_max_body_size 100M;
```

Comment:
Can we put a comment in here about "why" we suggest this setting?

Reply:
added